actors/builtin/miner/policy.go

```diff
@@ -130,10 +124,10 @@ type VestSpec struct {
 }
 
 var PledgeVestingSpec = VestSpec{
-	InitialDelay: abi.ChainEpoch(SecondsInYear / EpochDurationSeconds),    // 1 year, PARAM_FINISH
-	VestPeriod:   abi.ChainEpoch(SecondsInYear / EpochDurationSeconds),    // 1 year, PARAM_FINISH
-	StepDuration: abi.ChainEpoch(7 * SecondsInDay / EpochDurationSeconds), // 1 week, PARAM_FINISH
-	Quantization: SecondsInDay / EpochDurationSeconds,                     // 1 day, PARAM_FINISH
+	InitialDelay: abi.ChainEpoch(7 * abi.SecondsInDay / abi.EpochDurationSeconds),   // 1 week for testnet, PARAM_FINISH
+	VestPeriod:   abi.ChainEpoch(7 * abi.SecondsInDay / abi.EpochDurationSeconds),   // 1 week for testnet, PARAM_FINISH
+	StepDuration: abi.ChainEpoch(12 * abi.SecondsInHour / abi.EpochDurationSeconds), // 12 hours for testnet, PARAM_FINISH
+	Quantization: abi.SecondsInHour / abi.EpochDurationSeconds,                      // 1 hour for testnet, PARAM_FINISH
```

Comment:
This will mean we see 24x more expensive vesting than is representative. Set the quantization up to at least 6 hours, maybe higher. Consider setting the step duration to 1 day to leave room.

Reply:
got it, makes sense